dtfs-central-api/src/helpers/get-ukef-facility-ids.test.ts

```diff
@@ -1,11 +1,9 @@
 import { TfmFacility } from '@ukef/dtfs2-common';
-import { MOCK_FACILITY_SNAPSHOT } from '../../../__mocks__/mock-facility-snapshot';
 import { getUkefFacilityIds } from './get-ukef-facility-ids';
 
 const aFacility = ({ ukefFacilityId }: { ukefFacilityId: unknown }): TfmFacility =>
   ({
     facilitySnapshot: {
-      ...MOCK_FACILITY_SNAPSHOT,
```

Comment:
Moved this helper file from tfm-api to dtfs-central (this mock snapshot is a tfm-api object, so removed here and is not needed for the tests)